Library/Homebrew/cmd/update-report.rb

```diff
@@ -22,16 +22,23 @@ def update_report
     HOMEBREW_REPOSITORY.cd do
       analytics_message_displayed = \
         Utils.popen_read("git", "config", "--local", "--get", "homebrew.analyticsmessage").chuzzle
+      caskanalyticsmessage = \
+        Utils.popen_read("git", "config", "--local", "--get", "homebrew.cask.analyticsmessage").chuzzle
```

Comment:
I'd suggest `homebrew.caskanalyticsmessage` here

Reply:
Earlier ( https://github.com/Homebrew/brew/pull/4620#issuecomment-411760286 ) you suggested the current implementation. Could you confirm that you would now like it changed?

Reply:
Yes, please, sorry. I wasn't familiar with what the existing keys were at the time.